.gitignore

```diff
@@ -1,5 +1,5 @@
-/target/
+/target
```

Comment:
If we take out the rule below (`**/target/`) then I believe this actually should be `target/` after looking at the gitignore docs again

Reply:
It's exactly what `cargo new` puts in the new libraries.

Reply:
Eyup! I believe that's because `cargo new` sets it up for a single library, not a workspace. At least locally when I build `thruster/thruster` with this gitignore, all of the target dirs are not ignored.

From the docs it looks like `/target` will only apply to the base dir, whereas `target/` will apply to any dir named `target`. What was there before (which I _do_ think we should change!) was ignoring the base `target` dir as well as each `target` dir specifically one level down.

Reply:
Cargo workspaces have a shared `target` at the root.

Reply:
Right, and so do each of the individual projects. To avoid committing each of the individual project targets if they're built, you need to not limit it to the root target. As examples see: [tokio](https://github.com/tokio-rs/tokio/blob/master/.gitignore#L1), [tonic](https://github.com/hyperium/tonic/blob/master/.gitignore), [amethyst](https://github.com/amethyst/amethyst/blob/master/.gitignore#L6).

Reply:
@vorot93 rightly pointed out offline that any residual targets were leftover from before there was a workspace set up. After that change there is only one target and it's at the root so this works!